doc/releases/changelog-dev.md

```diff
@@ -384,29 +405,94 @@
+  `Observable`. This allows it to return valid generators from `SymbolicOp` and `CompositeOp` classes.
+  [(#3485)](https://github.com/PennyLaneAI/pennylane/pull/3485)
+
+* The `qml.equal` function has been extended to compare `Prod` and `Sum` operators.
+  [(#3516)](https://github.com/PennyLaneAI/pennylane/pull/3516)
+
+* `qml.purity` is added as a measurement process for purity
+  [(#3551)](https://github.com/PennyLaneAI/pennylane/pull/3551)
+
+* In-place inversion has been removed for qutrit operations in preparation for the
+  removal of in-place inversion.
+  [(#3566)](https://github.com/PennyLaneAI/pennylane/pull/3566)
+
+* Moved `qml.utils.sparse_hamiltonian` function to `~.Hamiltonian.sparse_matrix` method.
+  [(#3585)](https://github.com/PennyLaneAI/pennylane/pull/3585)
+
+* The `PauliSentence.operation()` method has been improved to avoid instantiating an `SProd` operator when
+  the coefficient is equal to 1.
+  [(#3595)](https://github.com/PennyLaneAI/pennylane/pull/3595)
+
+* Allow batching in all `SymbolicOp` operators, which include `Exp`, `Pow` and `SProd`.
+  [(#3597)](https://github.com/PennyLaneAI/pennylane/pull/3597)
+
+* `Sum` and `Prod` operations now have broadcasted operands.
+  [(#3611)](https://github.com/PennyLaneAI/pennylane/pull/3611)
+
+* Implemented the XYX single-qubit unitary decomposition.
+  [(#3628)](https://github.com/PennyLaneAI/pennylane/pull/3628)
+
+* All dunder methods now return `NotImplemented`, allowing the right dunder method (e.g. `__radd__`)
+  of the other class to be called.
+  [(#3631)](https://github.com/PennyLaneAI/pennylane/pull/3631)
+
+* The `qml.GellMann` operators now include their index when displayed.
+  [(#3641)](https://github.com/PennyLaneAI/pennylane/pull/3641)
+
+* All `Operator`'s input parameters that are lists are cast into vanilla numpy arrays.
+  [(#3659)](https://github.com/PennyLaneAI/pennylane/pull/3659)
+
 * Added `qml.ops.ctrl_decomp_zyz` to compute the decomposition of a controlled single-qubit operation given
   a single-qubit operation and the control wires.
   [(#3681)](https://github.com/PennyLaneAI/pennylane/pull/3681)
 
-*Next generation device API:*
-
-* The `apply_operation` single-dispatch function is added to `devices/qubit` that applies an operation
-  to a state and returns a new state.
-  [(#3637)](https://github.com/PennyLaneAI/pennylane/pull/3637)
+* `qml.pauli.is_pauli_word` now supports `Prod` and `SProd` operators, and it returns `False` when a
+  `Hamiltonian` contains more than one term.
+  [(#3692)](https://github.com/PennyLaneAI/pennylane/pull/3692)
 
-* The `preprocess` function is added to `devices/qubit` that validates, expands, and transforms a batch
-  of `QuantumTape` objects to abstract preprocessing details away from the device.
-  [(#3708)](https://github.com/PennyLaneAI/pennylane/pull/3708)
+* `qml.pauli.pauli_word_to_string` now supports `Prod`, `SProd` and `Hamiltonian` operators.
```

Comment:
Yes :thinking: This looks ok to me though, since we're highlighting two separate improvements - they just happen to come from the same PR.

Would you recommend to merge them into one?

Reply:
Nope! Just wanted to flag it. That's fine if we leave them separate